stable/percona-xtradb-cluster/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: percona-xtradb-cluster
-version: 0.3.0
+version: 0.3.1
```

Comment:
As this is new feature this should be 0.4.0

Reply:
Fixed in new commit.